Project2015To2017/Writing/ProjectWriter.cs

```diff
@@ -218,6 +218,11 @@ internal XElement CreateXml(Project project)
 						projectReferenceElement.Add(new XElement("Aliases", projectReference.Aliases));
 					}
 
+					if (projectReference.EmbedInteropTypes)
```

Comment:
Setting this property to its correct value didn't make it into this PR due to dependency on future PR. Why not add writing the value now (though never executed at the moment).